bedrock/mozorg/templates/mozorg/contribute/tasks/joy-of-coding.html

```diff
@@ -29,7 +29,7 @@
           </a></p>
         {% endif %}
         <h4>{{ _('Get involved') }}</h4>
-        {{ _('<a href="%s" id="get-involved" target="_blank">Join the Coding Community</a> and start collaborating with other Mozillians around the world.')|format('https://wiki.mozilla.org/Contribute/Coding') }}
+        {{ _('<a href="%s" id="get-involved" target="_blank" rel="noopener noreferrer">Join the Coding Community</a> and start collaborating with other Mozillians around the world.')|format('https://wiki.mozilla.org/Contribute/Coding') }}
```

Comment:
I think we can actually remove `target="_blank"` on this link, but we'll still need an l10n conditional:

```
{% set url = 'https://wiki.mozilla.org/Contribute/Coding' %}
{% if l10n_has_tag('target-links-noopener') %}
  {{ _('<a href="%s" id="get-involved">Join the Coding Community</a> and start collaborating with other Mozillians around the world.')|format(url) }}
{% else %}
  {{ _('<a href="%s" id="get-involved" target="_blank">Join the Coding Community</a> and start collaborating with other Mozillians around the world.')|format(url) }}
{% endif %}
```